File: mybatis-flex-spring/src/main/java/com/mybatisflex/test/Application.java
package com.mybatisflex.test;

import com.mybatisflex.core.audit.AuditManager;
import com.mybatisflex.core.audit.ConsoleMessageCollector;
import com.mybatisflex.core.audit.MessageCollector;
import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@MapperScan("com.mybatisflex.test.mapper")
public class Application {

    public static void main(String[] args) {
        //开启审计功能
        AuditManager.setAuditEnable(true);

//设置 SQL 审计收集器
        MessageCollector collector = new ConsoleMessageCollector();
        AuditManager.setMessageCollector(collector);
        SpringApplication.run(Application.class, args);
    }
}

File: mybatis-flex-spring-kotlin/src/main/kotlin/com/mybatisflux/Application.kt
package com.mybatisflux

import org.mybatis.spring.annotation.MapperScan
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
@MapperScan("com.mybatisflux.mapper")
class Application


fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
